pkgs/development/ocaml-modules/ppx_deriving_yaml/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildDunePackage, fetchFromGitHub, ppxlib, alcotest
+, ppx_deriving, yaml
+}:
+
+buildDunePackage rec {
+  pname = "ppx_deriving_yaml";
+  version = "0.1.0";
+
+  useDune2 = true;
+
+  minimumOCamlVersion = "4.07";
+
+  src = fetchFromGitHub {
+    owner = "patricoferris";
+    repo = "ppx_deriving_yaml";
+    rev = "v${version}";
+    sha256 = "1pdxvkm7hd4iikfzndacj2xymmwz8py07lrnyyn7jm1x0619j1qf";
+  };
```

Comment:
You can use `fetchurl` and download the source from there: https://github.com/patricoferris/ppx_deriving_yaml/releases/download/v0.1.0/ppx_deriving_yaml-v0.1.0.tbz

The released artifact prepared by the authors in particular includes correct version information.

Reply:
oh ok i thought fetchfromgithub was the preferred method

Reply:
As `fetchurl` does not have a `meta.homepage` attribute, you must define it for your derivation.